dash/frontend/src/app/modules/private/pages/external-auth-configuration/external-auth-configuration-create/external-auth-configuration-create.component.html

```diff
@@ -67,6 +67,13 @@ <h2 mat-dialog-title class="subnav-title">
               <input formControlName="oauthScopes" matInput>
             </mat-form-field>
           </p>
+          <p>
+            <mat-form-field>
+              <mat-label>Allowed User Domains (comma separated)</mat-label>
```

Comment:
is this a required field? (if not, maybe it should be to prevent insecure usage)

Reply:
It's not, I can set it to be